form/form/form.hpp

```diff
@@ -15,7 +15,7 @@ namespace form::experimental {
   class form_interface {
   public:
     form_interface(std::shared_ptr<phlex::testing::product_type_names> tm,
-                   const form::experimental::config::parse_config& config);
+                   const phlex::config::parse_config& phlex_config); // Accept phlex config
```

Comment:
Didn't we agree on a "phlex::testing..." namespace to indicate this is _not_ real phlex code, but only used to test some constructs?